SPECS/cloud-init/cloud-init.spec

```
@@ -1,17 +1,20 @@
+%define upstream_version_group 23.3.3
+%define package_version %(echo %{upstream_version_group} | cut -d. -f1-2)
+
 Summary:        Cloud instance init scripts
 Name:           cloud-init
-Version:        23.4.1
-Release:        4%{?dist}
+Epoch:          1
+Version:        %{package_version}
+Release:        1%{?dist}
```

Need to avoid collisions with the `.rpm` paths, they omit the epoch and we already published a `-1` release.
```suggestion
Release:        2%{?dist}
```

---

Agreed. Bumped release. Thanks!